go/oasis-node/cmd/common/consensus/consensus.go

```diff
@@ -104,6 +109,10 @@ func SignAndSaveTx(tx *transaction.Transaction) {
 
 	sigTx, err := transaction.Sign(signer, tx)
 	if err != nil {
+		if viper.GetString(cmdSigner.CfgSigner) == "ledger" && err.Error() == ledgerRejectTxErrorMsg {
```

Comment:
This seems like the wrong place to put it? It should go into the implementation of the ledger signer.

Reply:
Agreed. I've moved it to the ledger signer implementation.